pysilcam/plotting.py

```diff
@@ -29,20 +30,20 @@ def __init__(self):
     def plot(self, imc, imbw, times, d50_ts, vd_mean, display):
         '''Create plots from data'''
 
-        #Plot image in upper left axis
+        # Plot image in upper left axis
         ax = self.ax[0, 0]
-        if display==True:
+        if (display is True):
```

Comment:
```suggestion
        if display is True:
```
Note that if display can be either a bool or another type of variable (like a str) then this is what you should do (but isn't ideal programming imo). But if display is always a bool, then you can just use `if display:`.